[2022-10-26 13:37:53,924] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 13:37:53,942] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 13:37:53,942] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:37:53,942] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 13:37:53,943] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:37:53,966] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): deportistas_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 13:37:53,972] {standard_task_runner.py:52} INFO - Started process 469 to run task
[2022-10-26 13:37:53,976] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'deportistas_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmpes5ka4u1', '--error-file', '/tmp/tmp15w9gakl']
[2022-10-26 13:37:53,978] {standard_task_runner.py:80} INFO - Job 22: Subtask deportistas_raw_populate
[2022-10-26 13:37:54,092] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host b1681a16582a
[2022-10-26 13:37:54,242] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=deportistas_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 13:37:54,261] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 13:37:54,268] {dbapi.py:208} INFO - Running statement: COPY deportistas_raw FROM '/tmp/sample_data/deportista.csv' DELIMITER ',' CSV HEADER;
                                COPY deportistas_raw FROM '/tmp/sample_data/deportista2.csv' DELIMITER ',' CSV HEADER;
                        , parameters: None
[2022-10-26 13:37:57,773] {dbapi.py:216} INFO - Rows affected: 67706
[2022-10-26 13:37:57,803] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=deportistas_raw_populate, execution_date=20220901T000000, start_date=20221026T133753, end_date=20221026T133757
[2022-10-26 13:37:57,841] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 13:37:57,916] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-10-26 14:27:35,783] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:27:35,801] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:27:35,801] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:27:35,801] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 14:27:35,802] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:27:35,822] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): deportistas_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 14:27:35,826] {standard_task_runner.py:52} INFO - Started process 269 to run task
[2022-10-26 14:27:35,831] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'deportistas_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmp7p0a057h', '--error-file', '/tmp/tmptfomh0wi']
[2022-10-26 14:27:35,833] {standard_task_runner.py:80} INFO - Job 14: Subtask deportistas_raw_populate
[2022-10-26 14:27:35,902] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host d860ef10ab5f
[2022-10-26 14:27:36,016] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=deportistas_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 14:27:36,029] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 14:27:36,034] {dbapi.py:208} INFO - Running statement: COPY deportistas_raw FROM '/tmp/sample_data/deportista.csv' DELIMITER ',' CSV HEADER;
                                COPY deportistas_raw FROM '/tmp/sample_data/deportista2.csv' DELIMITER ',' CSV HEADER;
                        , parameters: None
[2022-10-26 14:27:38,494] {dbapi.py:216} INFO - Rows affected: 67706
[2022-10-26 14:27:38,511] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=deportistas_raw_populate, execution_date=20220901T000000, start_date=20221026T142735, end_date=20221026T142738
[2022-10-26 14:27:38,573] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:27:38,640] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-10-26 14:47:50,077] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:47:50,097] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 14:47:50,097] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:47:50,098] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 14:47:50,098] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:47:50,126] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): deportistas_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 14:47:50,131] {standard_task_runner.py:52} INFO - Started process 308 to run task
[2022-10-26 14:47:50,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'deportistas_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmpkh3tlsjo', '--error-file', '/tmp/tmpxcmch8lt']
[2022-10-26 14:47:50,137] {standard_task_runner.py:80} INFO - Job 14: Subtask deportistas_raw_populate
[2022-10-26 14:47:50,235] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host 4af2e895db31
[2022-10-26 14:47:50,382] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=deportistas_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 14:47:50,398] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 14:47:50,405] {dbapi.py:208} INFO - Running statement: COPY deportistas_raw FROM '/tmp/sample_data/deportista.csv' DELIMITER ',' CSV HEADER;
                                COPY deportistas_raw FROM '/tmp/sample_data/deportista2.csv' DELIMITER ',' CSV HEADER;
                        , parameters: None
[2022-10-26 14:47:52,777] {dbapi.py:216} INFO - Rows affected: 67706
[2022-10-26 14:47:52,792] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=deportistas_raw_populate, execution_date=20220901T000000, start_date=20221026T144750, end_date=20221026T144752
[2022-10-26 14:47:52,838] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:47:52,910] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-10-26 17:10:58,128] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 17:10:58,156] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-26 17:10:58,157] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 17:10:58,157] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-26 17:10:58,157] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 17:10:58,198] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): deportistas_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-26 17:10:58,204] {standard_task_runner.py:52} INFO - Started process 268 to run task
[2022-10-26 17:10:58,211] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'deportistas_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmpyfm9mtqe', '--error-file', '/tmp/tmp2e3g9whi']
[2022-10-26 17:10:58,213] {standard_task_runner.py:80} INFO - Job 14: Subtask deportistas_raw_populate
[2022-10-26 17:10:58,422] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host 553edc2a1daa
[2022-10-26 17:10:58,670] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=deportistas_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-26 17:10:58,689] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-26 17:10:58,696] {dbapi.py:208} INFO - Running statement: COPY deportistas_raw FROM '/tmp/sample_data/deportista.csv' DELIMITER ',' CSV HEADER;
                                COPY deportistas_raw FROM '/tmp/sample_data/deportista2.csv' DELIMITER ',' CSV HEADER;
                        , parameters: None
[2022-10-26 17:11:02,756] {dbapi.py:216} INFO - Rows affected: 67706
[2022-10-26 17:11:02,772] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=deportistas_raw_populate, execution_date=20220901T000000, start_date=20221026T171058, end_date=20221026T171102
[2022-10-26 17:11:02,835] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 17:11:02,906] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-10-29 18:31:54,002] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-29 18:31:54,020] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-29 18:31:54,020] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 18:31:54,020] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-10-29 18:31:54,020] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-29 18:31:54,043] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): deportistas_raw_populate> on 2022-09-01 00:00:00+00:00
[2022-10-29 18:31:54,049] {standard_task_runner.py:52} INFO - Started process 401 to run task
[2022-10-29 18:31:54,053] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_raw_data', 'deportistas_raw_populate', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/raw-dag.py', '--cfg-path', '/tmp/tmp8umxnhr7', '--error-file', '/tmp/tmptnz5071n']
[2022-10-29 18:31:54,055] {standard_task_runner.py:80} INFO - Job 16: Subtask deportistas_raw_populate
[2022-10-29 18:31:54,127] {task_command.py:369} INFO - Running <TaskInstance: load_raw_data.deportistas_raw_populate scheduled__2022-09-01T00:00:00+00:00 [running]> on host d3faad85a14a
[2022-10-29 18:31:54,241] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_raw_data
AIRFLOW_CTX_TASK_ID=deportistas_raw_populate
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-29 18:31:54,255] {base.py:68} INFO - Using connection ID 'postgres-default' for task execution.
[2022-10-29 18:31:54,260] {dbapi.py:208} INFO - Running statement: COPY oly.deportistas_raw FROM '/tmp/sample_data/deportista.csv' DELIMITER ',' CSV HEADER;
                                COPY oly.deportistas_raw FROM '/tmp/sample_data/deportista2.csv' DELIMITER ',' CSV HEADER;
                        , parameters: None
[2022-10-29 18:31:57,914] {dbapi.py:216} INFO - Rows affected: 67706
[2022-10-29 18:31:57,934] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=load_raw_data, task_id=deportistas_raw_populate, execution_date=20220901T000000, start_date=20221029T183154, end_date=20221029T183157
[2022-10-29 18:31:57,976] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-29 18:31:58,045] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
